src/test/management/apps.spec.ts

```diff
@@ -469,7 +469,7 @@ describe("App management", () => {
       const appCounts = 10;
       const expectedAppList = generateIosAppList(appCounts);
       const apiResponseAppList = expectedAppList.map((app) => {
-        const iosApp = { ...app };
+        const iosApp: any = { ...app };
```

Comment:
Would you mind leaving a TODO to clean this up?

Reply:
Done.